pkg/domain/domain.go

```diff
@@ -2256,6 +2256,22 @@ func (do *Domain) UpdateTableStatsLoop(ctx, initStatsCtx sessionctx.Context) err
 		},
 		"analyzeJobsCleanupWorker",
 	)
+	go func() {
```

Comment:
do.wg.Run(func(){
},"DeleteInternalSession init stats")

Reply:
+1, how to test this?